tools/bisector/bisector/bisector.py

```diff
@@ -5138,10 +5251,21 @@ def _main(argv):
 
     SHOW_TRACEBACK = args.debug
     service_hub = ServiceHub()
+
+    artifacts_service = None
     try:
-        service_hub.register_service('upload', ArtifactorialService())
-    except Exception as e:
-        info('Artifactorial upload will not be available: ' + str(e))
+        artifacts_service = ArtifactoryService()
+        info('Artifactory Service was initialized.')
+    except:
```

Comment:
bare `except:` are a very bad idea. They will catch any subclass of `BaseException`, which include things like `KeyboardInterrupt`. In this sort of context, the most general exception that should be caught is `Exception`

https://docs.python.org/3/library/exceptions.html#exception-hierarchy